tests/queries/0_stateless/02995_baseline_23_12_1.tsv

```diff
@@ -378,6 +378,7 @@ input_format_json_read_objects_as_strings	1
 input_format_json_try_infer_named_tuples_from_objects	1
 input_format_json_try_infer_numbers_from_strings	0
 input_format_json_validate_types_from_metadata	1
+input_format_json_ignore_key_case	0
```

Comment:
That's not right, this setting didn't exist in 23.12 version, please, don't change this file and add new setting to SettingsChangesHistory.h